contracts/Market.sol

```diff
@@ -389,26 +389,40 @@ contract Market is IMarket, Ownable, ERC721 {
 		return _bets[index].payout;
 	}
 
-
 	function _payout(uint256 index, uint8 result) internal virtual {
-		if (result == SCRATCHED) {
-			uint256 lay = _bets[index].payout - _bets[index].amount;
+		uint256 payout = _bets[index].payout;
+		uint256 amount = _bets[index].amount;
+		uint256 loan = payout - amount;
+
+		if (result == SCRATCHED) {				
```

Comment:
It would be good if this tidy-up was also applied to `_payout` on the collateralised market to keep them similar. Ideally it should inherit most of this code, actually...